frontend/frontend/src/components/Timer.vue

```diff
@@ -1,6 +1,7 @@
 <template>
 	<div>
-		<span>{{ time }}</span>
+		<span v-if="maxTime != '00:20:00'">{{ time }} out of {{ maxTime }}</span>
+		<span v-else>{{ time }}</span>
```

Comment:
In the v-else span, could we add the text "No time Limit!" next to the time? This way we can indicate to the user that the "easy" level means no time limit. What do you think?